src/python/py/models/builder.py

```diff
@@ -1770,11 +1795,15 @@ def make_model(self, input_path):
                 self.make_layer(self.layer_id, module)
                 self.layer_id += 1
 
+            elif module.__class__.__name__.endswith("GLMBlock") and self.layer_id < self.num_layers:
+                print(f"Reading decoder layer {self.layer_id}")
+                self.make_layer(self.layer_id, module)
+                self.layer_id += 1
+
             elif self.layer_id == self.num_layers and self.has_final_norm(module, model):
                 # SkipLayerNorm after last decoder layer (MatMul --> SkipLayerNorm)
                 print("Reading final norm")
                 self.make_layernorm(self.layer_id, module, skip=True, simple=self.layernorm_attrs["simple"], location="final_norm")
-
```

Comment:
The extra line space between the conditions helps make the different cases more readable. Can this line deletion be reverted?